### PR TITLE
Wasted event arguments

### PR DESCRIPTION
**Pull request by ghost**
### Description
This pull request adds changes that were made in #3557 but later reverted because they required a separate pr.
This includes adding the missing arguments `animGroup` and `animID`.

### CODE REVIEW

File: Client/mods/deathmatch/logic/CClientGame.cpp [hunk 1]
```diff
@@ -2638,7 +2638,7 @@ void CClientGame::AddBuiltInEvents()
     m_Events.AddEvent("onClientPlayerRadioSwitch", "", NULL, false);
     m_Events.AddEvent("onClientPlayerDamage", "attacker, weapon, bodypart", NULL, false);
     m_Events.AddEvent("onClientPlayerWeaponFire", "weapon, ammo, ammoInClip, hitX, hitY, hitZ, hitElement", NULL, false);
-    m_Events.AddEvent("onClientPlayerWasted", "", NULL, false);
+    m_Events.AddEvent("onClientPlayerWasted", "ammo, killer, weapon, bodypart, isStealth, animGroup, animID", NULL, false);
```
Comment (TracerDS) [inline Client/mods/deathmatch/logic/CClientGame.cpp]: Nullptr instead of NULL

File: Server/mods/deathmatch/logic/CGame.cpp [hunk 2]
```diff
@@ -1570,7 +1570,7 @@ void CGame::AddBuiltInEvents()
     m_Events.AddEvent("onPlayerQuit", "reason", NULL, false);
     m_Events.AddEvent("onPlayerSpawn", "spawnpoint, team", NULL, false);
     m_Events.AddEvent("onPlayerTarget", "target", NULL, false);
-    m_Events.AddEvent("onPlayerWasted", "ammo, killer, weapon, bodypart", NULL, false);
+    m_Events.AddEvent("onPlayerWasted", "ammo, killer, weapon, bodypart, isStealth, animGroup, animID", NULL, false);
```
Comment (TracerDS) [inline Server/mods/deathmatch/logic/CGame.cpp]: Nullptr instead of NULL

File: Server/mods/deathmatch/logic/CGame.cpp [hunk 3]
```diff
@@ -1602,7 +1602,7 @@ void CGame::AddBuiltInEvents()
     // Ped events
     m_Events.AddEvent("onPedVehicleEnter", "vehicle, seat, jacked", NULL, false);
     m_Events.AddEvent("onPedVehicleExit", "vehicle, reason, jacker", NULL, false);
-    m_Events.AddEvent("onPedWasted", "ammo, killer, weapon, bodypart", NULL, false);
+    m_Events.AddEvent("onPedWasted", "ammo, killer, weapon, bodypart, isStealth, animGroup, animID", NULL, false);
```
Comment (TracerDS) [inline Server/mods/deathmatch/logic/CGame.cpp]: Nullptr instead of NULL

**Review by TracerDS — APPROVED** [review]

**Review by FileEX — APPROVED** [review]

**Review by TracerDS — APPROVED** [review]

**Comment by Dutchman101** [comment]
Thanks

**Pull request merged**